src/warc2zim/converter.py

```diff
@@ -127,6 +127,20 @@ def __init__(self, args):
         with tempfile.NamedTemporaryFile(dir=self.output, delete=True) as fh:
             logger.debug(f"Confirming output is writable using {fh.name}")
 
+        # ensure zim file is creatable with the given name
+        try:
+            file_path = pathlib.Path(self.full_filename)
+            file_path.touch()
+            file_path.unlink()
+            logger.debug(
+                f"Confirming zim file can be created using name: {self.zim_file}"
+            )
+        except Exception:
+            logger.error(
+                f"Failed to create zim file with name: {self.zim_file}. Make sure the file name is valid."
```

Comment:
Idem

Reply:
Fixed in [00823a9](https://github.com/openzim/warc2zim/pull/234/commits/00823a9a4c0cfe42bb3b75c6d7ed450519933439)